tsconfig.json

```diff
@@ -2,6 +2,7 @@
     "compilerOptions": {
         "lib": ["ES2021"],
         // TODO: explore replacing the below setting with just target:ES2015
-        "downlevelIteration": true
-    }
+        "downlevelIteration": true,
+        "strict": true,
```

Comment:
@Mersho please:
1. move this to be the very first flag of the JSON file
2. I have NEVER told you to move the `tsc --version` outside of this PR. READ MY LATEST MESSAGES AGAIN TO TRY TO UNDERSTAND WHAT I SAID